pantheon-bundle/src/test/resources/validation-messages.properties

```diff
@@ -0,0 +1 @@
+module.icluded="Module cannot include another module"
```

Comment:
Ah... I see that this file is duplicated for testing purposes. I'd like to avoid that if possible. We might be able to use a symlink?